src/components/NftPreview/NftPreview.tsx

```diff
@@ -120,14 +126,19 @@ function NftPreview({
     if (shouldLiverender && token.media?.__typename === 'HtmlMedia') {
       return <NftDetailAnimation mediaRef={token} />;
     }
+
+    if (token.media?.__typename === 'UnknownMedia') {
+      return <FailedNftPreview onLoad={setContentIsLoaded} />;
+    }
```

Comment:
hmm so `UnknownMedia` does not necessarily mean the asset lacks URLs. the backend can still send URLs even for this type:
![image](https://user-images.githubusercontent.com/12162433/179650358-8e597674-2bd4-43f0-a290-5b15f76fbe3f.png)

so i don't think we need to show this 🤔 unless this fixes an issue?